ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -0,0 +1,134 @@
+''' EmailAgent
+
+  This agent reads a cache file ( cache.json ) which contains the aggregated information
+  of what happened to the elements of each site. After reading the cache file
+  ( by default every 30 minutes ) it sends an email for every site and then clears it.
+
+'''
+
+
+import os
+import json
+from DIRAC                                                       import S_OK, S_ERROR
+from DIRAC.Core.Base.AgentModule                                 import AgentModule
+from DIRAC.ResourceStatusSystem.Utilities                        import RssConfiguration
+from DIRAC.Interfaces.API.DiracAdmin                             import DiracAdmin
+
+__RCSID__ = '$Id: $'
+
+AGENT_NAME = 'ResourceStatus/EmailAgent'
+
+class EmailAgent( AgentModule ):
+
+  def __init__( self, *args, **kwargs ):
+
+    AgentModule.__init__( self, *args, **kwargs )
+    self.diracAdmin = DiracAdmin()
```

Comment:
please move this instantiation of `self.diracAdmin` in the `initialize` method: there are some cases where loading dirac objects inside the `__init__` of an agent leads to configuration errors